lib/vagrant-service-manager/command.rb

```diff
@@ -150,11 +150,11 @@ def print_all_provider_info(options = {})
 
           running_service_classes.each do |service_class|
             service = service_class.to_s.split('::').last.downcase
-            unless options[:script_readable] || KUBE_NAMES.include?(service)
+            unless options[:script_readable] || service == 'kubernetes'
```

Comment:
OOI, why is is better to spell 'kubernetes' multiple times instead of using a constant?